src/Core/Resources/DisplayStrings.resx

```diff
@@ -119,23 +119,30 @@
   </resheader>
   <data name="BoundingRectangleFormat" xml:space="preserve">
     <value>[l={0},t={1},w={2},h={3}]</value>
+    <comment>Do not translate: the coordinate specification of a bounding rectangle for an accessibility element.</comment>
   </data>
   <data name="HorizontalOrientation" xml:space="preserve">
     <value>Horizontal(1)</value>
+    <comment>A description of the visual orientation of an accessibility element.</comment>
   </data>
   <data name="LeftTopRightBottomFormat" xml:space="preserve">
     <value>[l={0},t={1},r={2},b={3}]</value>
+    <comment>Do not translate: the coordinate specification of the position of an accessibility element.</comment>
```

Comment:
Would it make sense to use full words here (for example, `left` instead of `l`) so that they can be translated? Same comment for the BoundingRectangleFormat string.

Reply:
It might – NVDA uses full words in its `location` property (closest analog).